demos/python_demos/smart_library_demo/smart_library_demo.py

```diff
@@ -0,0 +1,252 @@
+import sys, os
+import argparse
+import numpy as np
+import cv2 as cv
+import json
+
+sys.path.insert(0,'src/modules')
+import face_recognizer as fr 
+import book_recognizer as br
+
+sys.path.insert(0, 'src/infrastructure')
+from DynamicDatabase import *
+
+uknownID = -1
+count = 0
+
+def CreateArgparse():
+    parser = argparse.ArgumentParser(description='Smart Library Sample') 
+    parser.add_argument('-fr', type = str,
+                        dest = 'fRec', required=True, 
+                        help = 'Required. \
+                        Type of recognizer. \
+                        Available DNN face recognizer - DNNfr')
+    parser.add_argument('-m_rd', type = str, 
+                        dest = 'rdModel', required=True,  
+                        help = 'Required. Path to .xml file')        
```

Comment:
```suggestion
                        help = 'Required. Path to .xml file with a face reidentification model')
```
You should clarify what model is expected. The same goes for other keys.